src/core/dispatcher/Dispatcher.ts

```diff
@@ -1,146 +1,102 @@
 import JWEditor from '../JWEditor';
-import { ActionGenerator } from '../actions/ActionGenerator';
-import { PluginHandlers, PluginCommands } from '../JWPlugin';
 
-export type ActionHandler = (action: Action) => void;
-export type CommandIdentifier = string;
-export type HandlerToken = string;
-export type Handlers = Record<HandlerToken, ActionHandler>;
-export type DispatcherRegistry = Record<ActionIdentifier, Handlers>;
-export type DispatchFunction = (action: Action) => void;
+export type CommandFn = (...args: any) => void;
+export interface CommandHook {
+    id: string;
+    fn: CommandFn;
+}
+export interface Command extends CommandHook {
+    name?: string;
+    description?: string;
+}
+interface DipatcherCommand extends Command {
+    hooks: CommandFn[];
+}
```

Comment:
As discussed, I rened `hooks` by `handlers`.
Because command does not have handlers, I need to create this type to add the key.
How would you do otherwise?
Have the key `handlers` on the command itself?